docker-compose.yml

```diff
@@ -41,19 +41,19 @@ services:
     environment:
       AWS_ACCESS_KEY_ID: minioadmin
       AWS_SECRET_ACCESS_KEY: minioadmin
-      AWS_DEFAULT_REGION: us-east-1
+      AWS_DEFAULT_REGION: eu-west-2
 
-  generate-certificates:
-    image: docker.io/alpine/openssl@sha256:f46f31192b7e387f2999c6fa330cbbeff92519f9494fd1b4f9240f296ee53ad4
-    entrypoint: /mnt/scripts/generate-test-certificate
-    working_dir: /mnt/
-    volumes:
-      - source: ./resources/
-        target: /mnt/resources/
-        type: bind
-      - source: ./scripts/
-        target: /mnt/scripts/
-        type: bind
+  # generate-certificates:
```

Comment:
Had to comment this out

tried changing it to a ubuntu image

and changing the gen cert script to 

#!/bin/bash
# Generate a self-signed certificate for testing purposes.
set -xe

apt update
apt install -y mkcert
create_dir=resources/test/certs
mkdir -p ${create_dir}
pushd ${create_dir}
mkcert localhost
popd
cat ${create_dir}/localhost.pem ${create_dir}/localhost-key.pem > ./resources/test/nginx/localhost.pem


and then on the host machine
cd into resources/test/certs
and running mkcert -install

and confirmed that cert was installed in my trust store
on linux (/usr/local/share/ca-certificates/)
but it wasn't working properly and i was getting x509 errors when attempting to run terraform init.

and I wasn't sure how to disable the cert check in .terraformrc

Reply:
did u run `update-ca-certificates`

Reply:
I didn't


do you mean on my host machine?

sorry i think it's a bit confusing because my instructions are for local development.

more clear now that i've moved it to its own page